lib/cadet/course/group.ex

```diff
@@ -8,14 +8,14 @@ defmodule Cadet.Course.Group do
   alias Cadet.Accounts.User
 
```

Comment:
Are we moving this to the `Account` context?

Reply:
Based on the current contents and the Course context, I thought it's better to leave it.